tests/Unit/Services/VoucherEvaluator/VoucherEvaluatorTest.php

```diff
@@ -170,74 +264,34 @@ public function itCreditsWhenAChildIsUnderSchoolAge()
         $this->assertEquals(1, count($credits));
 
         // Check the correct credit type is applied.
-        $this->assertNotContains(self::CREDIT_TYPES['ChildIsUnderOne'], $credits, '');
-        $this->assertContains(self::CREDIT_TYPES['ChildIsUnderSchoolAge'], $credits, '');
+        $this->assertContains(self::CREDIT_TYPES['ChildIsBetweenOneAndPrimarySchoolAge'], $credits, '');
         $this->assertEquals(3, $evaluation->getEntitlement());
     }
 
     /** @test */
-    public function itCreditsWhenAChildIsUnderSecondarySchoolAge()
+    public function itDoesNotCreditWhenAChildisSecondarySchoolAge()
     {
-        // Make a Child under School Age.
-        $child = factory(Child::class, 'underSchoolAge')->make();
+        // Make a Secondary school child
+        $child = factory(Child::class, 'isSecondarySchoolAge')->make();
 
-        // Make standard evaluator
-        $evaluator = EvaluatorFactory::make('extended_age');
-        $evaluation = $evaluator->evaluate($child);
-        $credits = $evaluation["credits"];
+        $rulesMod = collect($this->rulesMods["credit-extended"]);
```

Comment:
Do we want credit extended mod here?

Reply:
it's a rulesMod element. I think that's ok?